aws_generated_data/commands/msk_eol.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from datetime import (
+    date,
+    datetime,
+    timedelta,
+)
+from pathlib import Path
+from typing import Any
+
+import requests
+import typer
+from bs4 import BeautifulSoup
+from pydantic import BaseModel, field_validator
+from typing_extensions import Annotated
+
+from aws_generated_data.utils import (
+    filter_items,
+    parse_date,
+    read_output_file,
+    write_output_file,
+)
+
+app = typer.Typer()
+log = logging.getLogger(__name__)
+
+
+class MskItem(BaseModel):
+    version: str
+    eol: date
+
+    @field_validator("version", mode="before")
+    def version_remove_asterisk(cls, value: str) -> str:
+        return value.strip(" ").rstrip("*")
```

Comment:
now this `rstrip("*")` is needed in all 3 places, should it belongs to parse logic and extract a similar method to `parse_date`?